src/helper.h

```diff
@@ -99,6 +99,12 @@ class Client: public RefCountable
     /// \param madeProgress whether the died helper(s) responded to any requests
     void handleFewerServers(bool madeProgress);
 
+    /// creates helper processes
+    virtual void openServers();
```

Comment:
To avoid legacy "server" terminology:

```suggestion
    virtual void openSessions();
```